### DISCUSSION TITLE
V3: x-html spelling mistake

### DISCUSSION THREAD

**Discussion by tao** [alpinejs/alpine#1449] [7. Bugs (archived)] [answered]
I don't know why `x-html` was removed... I use it all the time 😭 

The new website has a spelling mistake and the method `getHtml` and the function call `getHTML` are different:

```html
<!-- ✅ After -->
<!-- The above will now work with the following script added to the page: -->
<script>
    document.addEventListener('alpine:initializing', () => {
        Alpine.directive('html', (el, { expression }, { evaluateLater, effect }) => {
            let getHtml = evaluateLater(expression)

            effect(() => {
                getHTML(html => {
                    el.innerHTML = html
                })
            })
        })
    })
</script>
```

**Comment by ryangjchandler** [comment]
We @alpine-collective are working on a "toolkit" with helpers & directives. We've already got `x-html` ready for everyone relying on it, releasing soon!

**Comment by calebporzio** [comment, marked as answer]
@tao - I'll keep an eye on demand and maybe add it back in if it's useful enough to people. Also, fixed that docs thing. Thanks

**Reply by tao** [reply]
@calebporzio Great, I use it a lot on my website cause the whole paradigm (I understood) from frameworks like livewire, etc was generating static html partials on the server and replacing the inner HTML dynamically. So on my static website I generate partials for everything... like dropdowns, modals, navigation menus and I use simple Alpine x-html functions to replace them with @mouseenter and only load them when necessary... ✨

**Reply by maxsite** [reply]
x-html is very often used in AJAX. It would be good to leave it, even please compatibility.

**Reply by HugoDF** [reply]
A first party plugin probably makes sense

**Reply by maxsite** [reply]
Delete from Alpine X-HTML is stupid.
https://v3.ru.vuejs.org/ru/guide/template-syntax.html#%D1%82%D0%B5%D0%BA%D1%81%D1%82
